tools/integration_tests/run_e2e_tests.sh

```diff
@@ -42,8 +47,19 @@ fi
 if [ "$SKIP_NON_ESSENTIAL_TESTS_ON_PACKAGE" == true ]; then
   GO_TEST_SHORT_FLAG="-short"
   echo "Setting the flag to skip few un-important integration tests."
-  INTEGRATION_TEST_TIMEOUT=40m
+  INTEGRATION_TEST_TIMEOUT=50m
+  echo "Changing the integration test timeout to: $INTEGRATION_TEST_TIMEOUT"
+fi
+
+if [ "$RUN_TESTS_WITH_PRESUBMIT_FLAG" == true ]; then
```

Comment:
add a comment what we are trying to do here

Reply:
done